### PR TITLE
Remove resource-type select from batch upload

### LINKED ISSUE #1123: Resource type on Batch create
When adding new works via batch create, after uploading files, the UI prompts you to select the resource type for the work. With the implementation of multiple work types, this should instead be a selection between the available work types.

screen shot 2017-02-06 at 2 31 05 pm

### PR DESCRIPTION
**Pull request by crowesn**
Fixes #1123 

Remove :resource_type attribute and form select from batch creation form.

Changes proposed in this pull request:
* Add batch_create_job sufia override
* Override form js to remove resource_type select element
* Add specs

### CODE REVIEW

**Comment by scherztc** [comment]
My local environment is getting 4 failing test:

  3) BatchCreateJob#perform when visibility is passed sets public read access
     Failure/Error: let(:log)  { create(:batch_create_operation, user: user) }
     
     ArgumentError:
       Factory not registered: batch_create_operation

**Comment by crowesn** [comment]
No failures on my local.
```
Finished in 8 minutes 29 seconds (files took 9.18 seconds to load)
965 examples, 0 failures, 19 pending
```
Just did a rebase to resolve merge conflict.

File: spec/factories/operations.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+    trait :pending do
+      status CurationConcerns::Operation::PENDING
+    end
+
+    trait :successful do
+      status CurationConcerns::Operation::SUCCESS
+    end
+
+    factory :batch_create_operation, class: Sufia::BatchCreateOperation do
```
Comment (crowesn) [inline spec/factories/operations.rb]: Here's the factory from your failing tests.

Comment (crowesn) [inline spec/factories/operations.rb]: @scherztc Here's the factory from your failing test. Is this file not in your env?

**Pull request merged**